cobc/scanner.l

```diff
@@ -305,6 +307,26 @@ static void *	copy_literal (cb_tree l);
 	}
 }
 
+<*>{AREA_A}[[:space:]]*/[^[:alnum:][:space:]#] {
+	/* We exclude the `#` from the trail to avoid considering a subsequent
+	   `#line` as a punctuation.  Another alternative may be to add a
+	   mandatory space at the end of the pattern, as pre-processed lines are
+	   usually (?) prefixed with a space (does this hold for every source
+	   format?). */
+	/* <*>\n"#AREA_A"[[:space:]]*[ ]/[^[:alnum:][:space:]] { */
+	count_lines (yytext + 9); /* skip "\n#area_a\n" */
+	if (cobc_in_procedure) {
+		(void) cb_verify (cb_loose_area_a, _("punctuation in Area A"));
+	}
+}
+
+<*>{AREA_A} {
+	if (cobc_in_procedure) {
+		last_token_is_dot = 1; /* always assume we've just seen a dot. */
```

Comment:
Well… as of now any area A should either follow a dot or a "missing dot". `last_token_is_dot` is used to classify integer literals as they may be section or paragraph names; so maybe it may just need to be renamed?

Reply:
Renaming that (before the actual PR) would be possible, bot "period, then number" is also a level number. Not sure about the name yet.